app/Views/CoaBooking/CaseSearch.cshtml

```diff
@@ -257,11 +257,25 @@
                         {
                             <div class="preliminary_questions preliminary_questions--Chambers">
                                 <div class="row search-info-row">
-                                    <div class="col-md-6">
+                                    <div class="col-md-12">
                                         <label class="application_type can-wrap">
                                             Please select the application type(s) for this chambers hearing.
                                         </label>
-                                        @Html.DropDownListFor(x => x.SelectedApplicationTypes, new MultiSelectList(Model.ChambersApplicationTypes, "HearingTypeID", "ApplicationTypeName", Model.SelectedApplicationTypes))
+
+                                        @{
+                                            var availableDates = Json.Serialize(Model.ChambersApplicationTypes);
+                                            var selectedTypesJson = Json.Serialize(Model.SelectedApplicationTypes);
+                                            var applicationTypeSelectDisabled = Json.Serialize(@Model.Step3Complete);
+                                        }
+
+                                        <div id="VueApplicationTypeSelect">
+                                            <application-type-select
+                                                :options="@availableDates.ToString()"
+                                                :initial-selection="@selectedTypesJson.ToString()"
+                                                :disabled="@applicationTypeSelectDisabled"
+                                                v-on:select="onApplicationTypeChange"
+                                            />
+                                        </div>
```

Comment:
That works just fine.  It might be slightly preferable to use the `string` type explicitly instead of `var` just for readability because `var` looks like JavaScript and `string` looks like C#.

```
@{
    string availableDates = Json.Serialize(Model.ChambersApplicationTypes).ToString();
    string selectedTypesJson = Json.Serialize(Model.SelectedApplicationTypes).ToString();
    string applicationTypeSelectDisabled = Json.Serialize(@Model.Step3Complete).ToString();
}

<div id="VueApplicationTypeSelect">
    <application-type-select
        :options="@availableDates"
        :initial-selection="@selectedTypesJson"
        :disabled="@applicationTypeSelectDisabled"
        v-on:select="onApplicationTypeChange"
    />
</div>
```

I also moved "ToString()" up into the variable declarations.

Reply:
Thanks! That cleans it up nicely